### ISSUE TITLE
Schema update fails if table-name is a reserved word

### ISSUE THREAD

**Issue by maximilian-walter** [jsor/doctrine-postgis#19]
The schema update fails if the table-name is a reserved word, because the table-name will be quoted by Doctrine and the select-query which returns the column-information fails. So Doctrine thinks he has to change the "geometry_type" (from a empty string to the configured value), which is not supported (see #5).

The table-name was in my case `user`, which will be converted to `"user"`, which results in the following query:

````
SELECT coord_dimension, srid, type FROM geometry_columns WHERE f_table_name = '"user"' AND f_geometry_column = 'coordinates';
````

https://github.com/jsor/doctrine-postgis/blob/c8e19a53fb371596745ffc3065f3a256b2ed43b7/src/Schema/SchemaManager.php#L93

I'm new to Doctrine and I have no idea how to safely unquote the table-name, Is ``trim($table, '"');`` sufficient, or is there a better way?

I'm using PostgreSql 9.5 and the most current Doctrine-version.

**Comment by jsor** [comment]
I've commited a patch in a2ae238fafecc71ca8e8f32d6a0ff77be31e5aa8. Could you try the reserved-words branch (`composer require jsor/doctrine-postgis dev-reserved-words`) and report back if it fixes the issue.

**Comment by maximilian-walter** [comment]
Thank you for your fast response! The patch fixes the problem for me.

**Comment by jsor** [comment]
Thanks for reporting back! Fix released in [v1.1.2](https://github.com/jsor/doctrine-postgis/releases/tag/v1.1.2).
